ggshield/cmd/secret/scan/repo.py

```diff
@@ -13,11 +14,17 @@
 from ggshield.core.cache import Cache
 from ggshield.core.config import Config
 from ggshield.core.scan import ScanContext, ScanMode
-from ggshield.core.utils import REGEX_GIT_URL
 from ggshield.utils.git_shell import git
 from ggshield.verticals.secret.repo import scan_repo_path
 
 
+# Source: https://github.com/jonschlinkert/is-git-url MIT LICENSE
```

Comment:
Even if we only use this regex here, it is very generic and could be used in general code, maybe we should put it in `utils.git_shell` / `utils.git`, no?

:label: 1

Reply:
I'd rather not because I am not fond of this regex: it prevents commands like `ggshield secret scan repo https://github.com/GitGuardian/ggshield`. One must use `ggshield secret scan repo https://github.com/GitGuardian/ggshield.git`.